docs/blueprints/EP036.rst

```diff
@@ -0,0 +1,237 @@
+:EP: 36
+:Title: Openflow multi-table pipeline processing
+:Authors:
+    - Aldo Ortega <aortegay AT fiu DOT edu>
+    - Vinicius Arcanjo <vindasil AT fiu DOT edu>
+    - Italo Valcy <idasilva AT fiu DOT edu>
+
+:Created: 2023-05-04
+:Updated: 2023-10-04
+:Kytos-Version: 2022.3
+:Status: Draft
+
+***************************************************

```

Comment:
I was going to read the blueprint again, I just noticed on GitHub it's not rendering the .rst markeup, I'm not sure if it's because of extra `***` here in this table, but can you try to identify the issue? https://github.com/kytos-ng/kytos/blob/blueprint/pipeline/docs/blueprints/EP036.rst

![20230412_100018](https://user-images.githubusercontent.com/1010796/231465242-6cf370fb-5cb6-4387-920f-640254f30ea0.png)

If you open any of the blueprint links it's rending without problems, for instance EP033 https://github.com/kytos-ng/kytos/blob/blueprint/pipeline/docs/blueprints/EP033.rst

Reply:
Although we don't have a linter yet, looks like it might be related to line 145 @Alopalao, it still not rendering properly. Tiny trivial detail but later on if you can try to fix it, that way when we add this blueprint to the docs it'll render correctly

```
❯ restructuredtext-lint EP036.rst 
SEVERE EP036.rst:145 Title level inconsistent:
```

Reply:
Yeah, my IDE was not catching that, fixed.

Reply:
Cool. Let's close this thread.